values.yaml

```diff
@@ -0,0 +1,30 @@
+redis:
+  enabled: true
+  auth:
+    enabled: false
+  architecture: standalone
+  master:
+    persistence:
+      enabled: true
+      size: 1Gi  # Adjust the size as needed
+
+onetimesecret:
+  image: 
+    repository: onetimesecret/onetimesecret
+    tag: latest
+  service:
+    type: LoadBalancer
+    port: 3000
+  env:
+    REDIS_URL: "redis://onetimesecret-redis-master.default.svc.cluster.local:6379/0"
+    COLONEL: "admin@example.com"
+    HOST: "onetimesecret-onetimesecret.default.svc.cluster.local:3000"
```

Comment:
This value needs to be updated to the hostname or the correct namespace. I wonder if @delano can easily detect this in the code so we don't need it as a variable for the app.

Reply:
I think I have found a way to update the URLs with correct namespaces. I'm overwriting the values dynamically from deployment.yaml template.